src/Middleware/DocBlockAnnotations.php

```diff
@@ -86,7 +86,7 @@ private function derivePropertyDetailsFromReflectionProperty(\ReflectionProperty
             $type = substr($type, 0, -2);
         }
 
-        $nullable = stripos('|' . $type . '|', '|null|') !== false;
+        $nullable = stripos($docBlock, '|null') !== false;
```

Comment:
To calculate nullable here the `|null` string is searched in the whole DocBlock line.

Reply:
As mentioned in the above.